completions/bun.zsh

```diff
@@ -1,350 +1,752 @@
+        '--timeout[Set the per-test timeout in milliseconds, default is 5000.]:timeout' \
+        '--update-snapshots[Update snapshot files]' \
+        '--rerun-each[Re-run each test file <NUMBER> times, helps catch certain bugs]:rerun' \
+        '--only[Only run tests that are marked with "test.only()"]' \
+        '--todo[Include tests that are marked with "test.todo()"]' \
+        '--coverage[Generate a coverage profile]' \
+        '--bail[Exit the test suite after <NUMBER> failures. If you do not specify a number, it defaults to 1.]:bail' \
+        '--test-name-pattern[Run only tests with a name that matches the given regex]:pattern' \
+        '-t[Run only tests with a name that matches the given regex]:pattern' &&
+        ret=0
+
+    case $state in
+    file)
+        _bun_test_param_script_completion
+
+        ;;
+    config)
+        _files
+
+        ;;
+
+    esac
+
+}
+
+_bun() {
+    zstyle ':completion:*:*:bun:*' group-name ''
+    zstyle ':completion:*:*:bun-grouped:*' group-name ''
+
+    zstyle ':completion:*:*:bun::descriptions' format '%F{green}-- %d --%f'
+    zstyle ':completion:*:*:bun-grouped:*' format '%F{green}-- %d --%f'
+    # zstyle ':completion:*:options' list-colors '=^(-- *)=34'
+
+    local program=bun
+    typeset -A opt_args
+    local curcontext="$curcontext" state line context
+
+    # ---- Command:
+    _arguments -s \
+        '1: :->cmd' \
+        '*: :->args' &&
+        ret=0
+
+    case $state in
+    cmd)
+        local -a scripts_list
+        IFS=$'\n' scripts_list=($(SHELL=zsh bun getcompletes i))
+        scripts="scripts:scripts:(($scripts_list))"
+        IFS=$'\n' files_list=($(SHELL=zsh bun getcompletes j))
+
+        main_commands=(
+            'run\:"Run JavaScript with Bun, a package.json script, or a bin" '
+            'test\:"Run unit tests with Bun" '
+            'x\:"Install and execute a package bin (bunx)" '
+            'repl\:"Start a REPL session with Bun" '
+            'init\:"Start an empty Bun project from a blank template" '
+            'create\:"Create a new project from a template (bun c)" '
+            'install\:"Install dependencies for a package.json (bun i)" '
+            'add\:"Add a dependency to package.json (bun a)" '
+            'remove\:"Remove a dependency from package.json (bun rm)" '
+            'update\:"Update outdated dependencies & save to package.json" '
+            'link\:"Link an npm package globally" '
+            'unlink\:"Globally unlink an npm package" '
+            'pm\:"More commands for managing packages" '
+            'build\:"Bundle TypeScript & JavaScript into a single file" '
+            'upgrade\:"Get the latest version of bun" '
+            'help\:"Show all supported flags and commands" '
+        )
+        main_commands=($main_commands)
+        _alternative "$scripts" "args:command:(($main_commands))" "files:files:(($files_list))"
+
+        ;;
+    args)
+        case $line[1] in
+        add)
```

Comment:
You can change it to `add|a)` to also autocomplete for the alias like how it is in the [bash completions](https://github.com/oven-sh/bun/blob/main/completions/bun.bash#L135).
Other ones are `create|c` `install|i` and `remove|rm`.

Reply:
Fixed. thanks.